claims.go

```diff
@@ -108,10 +117,14 @@ func validateBoundClaims(logger log.Logger, boundClaims, allClaims map[string]in
 		}
 
 		found := false
+
+	scan:
 		for _, v := range expVals {
-			if actValue == v {
-				found = true
-				break
+			for _, av := range actVals {
+				if av == v {
```

Comment:
Will this match if one or both are an interface with the same underlying value? Not sure if that's what's happening here.

Reply:
Actually, I see the tests prove that it does match when it's supposed to. I'll resolve this!